gradle.properties

```diff
@@ -2,10 +2,10 @@
 org.gradle.jvmargs=-Xmx1G
 # Fabric Properties
 # check these on https://fabricmc.net/use
-minecraft_version=1.16.3
-yarn_mappings=1.16.3+build.11
-loader_version=0.9.3+build.207
+minecraft_version=1.16.5
+yarn_mappings=1.16.5+build.5
+loader_version=0.11.2
 # Mod Properties
-mod_version=1.0.11
+mod_version=1.0.12
 maven_group=com.ddwhm.jesen
-archives_base_name=imblockerfabric
\ No newline at end of file
+archives_base_name=imblockerfabric
```

Comment:
not neccessary

Reply:
I did not include this before, although may be because i added the context.